src/objdictgen/ui/subindextable.py

```diff
@@ -39,16 +39,12 @@
 
 RW = ["Read Only", "Write Only", "Read/Write"]
 RO = ["Read Only", "Read/Write"]
-RW_ACCESS_LIST = ",".join(RW)
```

Comment:
Are these removals also due to wx no longer needing the these vars to be comma separated strings?

Reply:
Yes, see comment here https://github.com/Laerdal/python-objdictgen/pull/2#discussion_r1405524995

These cs strings were only used in the GridCellChoiceEditor constructors and I therefore removed them.